how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Quick Start with TiDB
+summary: Learn how to quick start your first TiDB cluster and try out the features.
+category: how-to
+---
+
+# Quick Start with TiDB
+
+This document describes how to try out TiDB, a distributed SQL database. You can choose any of the following three methods to start your first TiDB cluster:
+
+- [Try out TiDB with one click using TiDB-Wasm](#try-out-tidb-with-one-click-using-tidb-wasm)
+- [Deploy a local test environment using TiUP Playground](#deploy-a-local-test-environment-using-tiup-playground)
+- [Deploy a quasi-production test environment on a single machine using TiUP cluster](#deploya-quasi-production-test-environment-on-a-single-machine-using-tiup-cluster)
+
+> **Note:**
+>
+> It is not recommended to deploy TiDB for production environments with the above three methods. Instead, you can [deploy TiDB using TiUP](/how-to/deploy/orchestrated/tiup.md).
+
+## Try out TiDB with one click using TiDB-Wasm
+
+- Scenario: Have a first taste on the basic features of TiDB, such as syntax and compatibility.
+- Time required: Instant
+
+TiDB-Wasm is a TiDB cluster running in the browser. You can use TiDB simply by opening a web page and perform basic operations such as executing SQL statements and verifying compatibility.
+
+Click to [try TiDB-Wasm](https://tour.pingcap.com). The TiDB cluster will be built in the memory, expectedly in around 10 seconds.
+
+## Deploy a local test environment using TiUP Playground
+
+- Scenario: Quickly deploy a TiDB cluster on a local Mac or Linux OS machine. You can see the basic architecture of TiDB and how TiDB, TiKV, PD, monitoring, and other components run in a TiDB cluster.
+- Time required: 1 minute
+
+As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB instances, 3 TiKV instances, and 3 PD instances. With TiUP Playground, you can quickly build the above test cluster by taking the following steps:
+
+1. Download and install TiUP:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. Declare the global environment variable:
+
+    {< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ``` 
+
+    > **Note:**
+    >
+    > After the installation, TiUP gives the absolute path of the corresponding `profile` file. You need to modify the above `source` command according to the path.
```

Comment:
```suggestion
    > **Note:**
    >
    > After the installation, TiUP gives the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.
```
Users might execute the command before they see this note, so I suggest moving this note above the command.

Reply:
gives -> displays